kademlia/network.py

```diff
@@ -61,7 +61,7 @@ def refreshTable(self):
         for id in self.protocol.getRefreshIDs():
             node = Node(id)
             nearest = self.protocol.router.findNeighbors(node, self.alpha)
-            spider = NodeSpiderCrawl(self.protocol, node, nearest, self.ksize, self.alpha)
+            spider = NodeSpiderCrawl(self.protocol, node, nearest)
```

Comment:
Hey - not sure why this line has changed - `NodeSpiderCrawl` uses `SpiderCrawl`'s constructor which requires these params.